pkg/core/account/accounts.go

```diff
@@ -162,6 +171,13 @@ func (a *Accounts) readFromReadSeeker(reader io.ReadSeeker) (n int, err error) {
 		a.setWithoutLocking(accountID, pool)
 	}
 
+	var reused bool
+	if err = binary.Read(reader, binary.LittleEndian, &reused); err != nil {
+		return n, errors.Wrap(err, "unable to read reused flag")
```

Comment:
Can you please use `ierrors` here instead?

Reply:
Only for the added line, there rest should be replaced by my following PR anyway.